### PR TITLE
feat: black background in activity and profile page

### LINKED ISSUE #1458: put with the black background the activity and profile page

### PR DESCRIPTION
**Pull request by Jaxiii**
## Changes

- CpTheme changed to dark on app.dar file, to override SystemOverlay on activity and profile screens
- Removed _Wrapper with FadeGradient, no longer needed.
- CpColors applied to Widgets where Colors used to be.
- Background and Text colors switched to CpColors dark theme.

## Related issues

Fixes #1458 

<details>
<summary>Screenshots</summary>
<br>

sc1
sc2
sc3
sc4

</details>

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [x] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

File: packages/espressocash_app/lib/features/profile/widgets/danger_section.dart [hunk 1]
```diff
@@ -22,8 +22,9 @@ class DangerSection extends StatelessWidget {
             label: context.l10n.signOut,
             onPressed: () => _handleLogoutPressed(context),
             color: CpColors.primaryColor,
```
Comment (ookami-kb) [inline packages/espressocash_app/lib/features/profile/widgets/danger_section.dart:24]: This color is wrong. Text color is `E85C30` and chevron color is `F5734A`.

File: packages/espressocash_app/lib/ui/app_bar.dart [hunk 2]
```diff
@@ -79,6 +80,9 @@ class _CpAppBarState extends State<CpAppBar> {
 
     return AppBar(
       title: widget.title,
+      titleTextStyle: Theme.of(context).textTheme.labelLarge?.copyWith(
+            color: CpColors.lightGreyBackground,
```
Comment (ookami-kb) [inline packages/espressocash_app/lib/ui/app_bar.dart:84]: It's breaking webview screens, e.g. transaction details screen. It should probably implement some dark theme as well:

**Review by ookami-kb — CHANGES_REQUESTED** [review]

File: packages/espressocash_app/lib/features/profile/widgets/danger_section.dart (hunk 1 above)
Comment (Jaxiii) [reply packages/espressocash_app/lib/features/profile/widgets/danger_section.dart]: Fixed at [359fd3a](https://github.com/espresso-cash/espresso-cash-public/pull/1461/commits/359fd3a2b3c4755698922053729b781ac844dc91)

File: packages/espressocash_app/lib/ui/app_bar.dart (hunk 2 above)
Comment (Jaxiii) [reply packages/espressocash_app/lib/ui/app_bar.dart]: Removed titleTextStyle from AppBar and applied theme to webviews in [2420692](https://github.com/espresso-cash/espresso-cash-public/pull/1461/commits/2420692a57c0b0623aec623a0a1e2eac6c44bf52)

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**